net-misc/youtube-viewer/youtube-viewer-3.2.7.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils gnome2-utils perl-module
+
+DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
+HOMEPAGE="https://trizenx.blogspot.com/2012/03/gtk-youtube-viewer.html"
+SRC_URI="https://github.com/trizen/youtube-viewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk"
+
+RDEPEND="
+	dev-perl/Data-Dump
+	dev-perl/JSON
+	dev-perl/libwww-perl[ssl]
+	dev-perl/Term-ReadLine-Gnu
+	dev-perl/LWP-Protocol-https
+	virtual/perl-Encode
+	virtual/perl-File-Path
+	virtual/perl-File-Spec
+	virtual/perl-Getopt-Long
+	virtual/perl-Scalar-List-Utils
+	virtual/perl-Term-ANSIColor
+	virtual/perl-Term-ReadLine
+	virtual/perl-Text-ParseWords
+	virtual/perl-Text-Tabs+Wrap
+	gtk? (
+		dev-perl/File-ShareDir
+		>=dev-perl/Gtk2-1.244.0
+		virtual/freedesktop-icon-theme
+		x11-libs/gdk-pixbuf:2[X,jpeg]
+	)
+	|| ( media-video/ffmpeg[openssl] media-video/ffmpeg[gnutls] )
+	|| ( media-video/mpv media-video/mplayer media-video/vlc gtk? ( media-video/smplayer ) )"
+DEPEND="dev-perl/Module-Build"
+
+SRC_TEST="do"
+
+src_prepare() {
+	perl-module_src_prepare
+}
+
+# build system installs files on "perl Build.PL" too
+# do all the work in src_install
+src_configure() { :; }
+src_compile() { :; }
+
+src_install() {
+	local myconf
+	if use gtk ; then
+		myconf="--gtk-youtube-viewer"
+	fi
+	perl-module_src_configure
```

Comment:
Why are these configure steps performed in install instead of configure? Seems weird.